### PR TITLE
feat: added fingerprints, bump apify version

### LINKED ISSUE #86: Update crawlers with fingerprints
Add usage of the newly added fingerprints to the templates when the corresponding library versions are released.

### PR DESCRIPTION
**Pull request by petrpatek**
closes  #86

### CODE REVIEW

File: templates/project_playwright_crawler/main.js [hunk 1]
```diff
@@ -29,6 +29,11 @@ Apify.main(async () => {
             // We don't have 'stealth' for Playwright yet.
             // Try using Firefox, it is naturally stealthy.
         },
+        browserPoolOptions: {
+            // This allows browser to be more effective against antis-craping protections.
```
Comment (metalwarrior665) [inline templates/project_playwright_crawler/main.js:33]: You have typo also in Puppeteer

**Review by metalwarrior665 — CHANGES_REQUESTED** [review]

**Comment by petrpatek** [comment]
Removed jest environment node due to this issue with jest: https://stackoverflow.com/questions/69809368/jest-cant-find-abortcontroller-in-node-v16-13-0

File: templates/project_puppeteer_crawler/main.js [hunk 2]
```diff
@@ -26,6 +26,11 @@ Apify.main(async () => {
             useChrome: true,
             stealth: true,
```
Comment (mnmkng) [inline templates/project_puppeteer_crawler/main.js:27]: I thought fingerprints would completely replace `stealth`. How do those two work together? Should we add `stealth` to Playwright as well?

Comment (petrpatek) [reply templates/project_puppeteer_crawler/main.js:27]: Stealth masks headless chrome to look like not automated chrome by flipping a few navigator properties. Fingerprints change already existing properties to provide higher entropy in the target browser.

Comment (mnmkng) [reply templates/project_puppeteer_crawler/main.js:27]: Thanks, but the question still stands: Do we need stealth for Playwright? Also, do we need to update the existing stealth as it's pretty outdated?

Comment (petrpatek) [reply templates/project_puppeteer_crawler/main.js:27]: We should add the argument behind `hideWebDriver` to the default launch arguments for chrome/chromium and then deprecate the stealth completely.

File: templates/example_puppeteer_single_page/package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     "version": "0.0.1",
     "description": "This is an example of an Apify actor.",
     "dependencies": {
-        "apify": "^2.0.7",
+        "apify": "^2.2.0",
```
Comment (B4nan) [inline templates/example_puppeteer_single_page/package.json:6]:
```suggestion
        "apify": "^2.3.2"
```

File: templates/example_typescript/package.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
     "version": "0.0.1",
     "description": "This is an example of an Apify actor.",
     "dependencies": {
-        "apify": "^2.0.7"
+        "apify": "^2.2.0"
```
Comment (B4nan) [inline templates/example_typescript/package.json:6]:
```suggestion
        "apify": "^2.3.2"
```

File: templates/project_cheerio_crawler/package.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
     "version": "0.0.1",
     "description": "This is a boilerplate of an Apify actor.",
     "dependencies": {
-        "apify": "^2.0.7"
+        "apify": "^2.2.0"
```
Comment (B4nan) [inline templates/project_cheerio_crawler/package.json:6]:
```suggestion
        "apify": "^2.3.2"
```

File: templates/project_empty/package.json [hunk 6]
```diff
@@ -3,7 +3,7 @@
     "version": "0.0.1",
     "description": "This is a boilerplate of an Apify actor.",
     "dependencies": {
-        "apify": "^2.0.7"
+        "apify": "^2.2.0"
```
Comment (B4nan) [inline templates/project_empty/package.json:6]:
```suggestion
        "apify": "^2.3.2"
```

File: templates/project_playwright_crawler/main.js [hunk 7]
```diff
@@ -29,6 +29,11 @@ Apify.main(async () => {
             // We don't have 'stealth' for Playwright yet.
             // Try using Firefox, it is naturally stealthy.
         },
+        browserPoolOptions: {
+            // This allows browser to be more effective against anti-scraping protections.
+            // If you are having performance issues try turning this off.
+            useFingerprints: true
```
Comment (B4nan) [inline templates/project_playwright_crawler/main.js:35]:
```suggestion
            useFingerprints: true,
```

File: templates/project_playwright_crawler/package.json [hunk 8]
```diff
@@ -3,7 +3,7 @@
     "version": "0.0.1",
     "description": "This is a boilerplate of an Apify actor.",
     "dependencies": {
-        "apify": "^2.0.7",
+        "apify": "^2.2.0",
```
Comment (B4nan) [inline templates/project_playwright_crawler/package.json:6]:
```suggestion
        "apify": "^2.3.2"
```

File: templates/project_puppeteer_crawler/main.js (hunk 2 above)
Comment (B4nan) [inline templates/project_puppeteer_crawler/main.js:27]:
```suggestion
```

we can remove this now as its done automatically in the browser pool

**Review by B4nan — COMMENTED** [review]
i'll merge this and finalize missing bits if needed in a separate PR

**Pull request merged**